packages/contracts/deploy/helpers.ts

```diff
@@ -321,14 +321,16 @@ export async function populatePluginRepo(
 
     const placeholderSetup = await getContractAddress('PlaceholderSetup', hre);
 
-    const emptyMetadata = ethers.utils.hexlify(ethers.utils.toUtf8Bytes(''));
+    const emptyJsonObject = ethers.utils.hexlify(
+      ethers.utils.toUtf8Bytes('{}')
```

Comment:
Passing `'{}'` instead of `{}` is necessary to prevent the `error EmptyReleaseMetadata()` to be thrown in the [`PluginRepo` contract](https://github.com/aragon/osx/blob/40f342b5906681e0050ec8bb984fc331070b932a/packages/contracts/src/framework/plugin/repo/PluginRepo.sol#L202-L204) (see the [ci pipeline error too]( https://github.com/aragon/osx/actions/runs/7063562315/job/19229813650#step:7:1563))